### ISSUE TITLE
Stability oft scol position, when changing the width.

### ISSUE THREAD

**Issue by tacruc** [Mikescops/vue-virtual-grid#6]
First oft all thanks for the lib. I just tried it in the demo page and noticed. That if I rotate my phone, from vertical to horizontal I not a single picture I have Seen before.
E.g. scrolling to section header 4 and rotating I had to scroll Quote a bit to be back at the header.

If it is possible I think it would be nice if the top left picture would stay in the top left field shown when changing the width. For languages which are read from right to legt I think it should be the top right picture which stays stable in the screen.

**Comment by Mikescops** [comment]
Hello,
Thanks for reporting this. It's interesting indeed, not sure how i can do this but I will try to solve this for sure.

**Comment by Mikescops** [comment]
I made a few tests earlier today, and i could not find a proper solution as of now without degrading performances.

**Comment by tacruc** [comment]
So then lets keep the performant way for now.

**Comment by MentalGear** [comment]
Indeed, when changing page window width, items are drawn upwards or downwards. It's just a nuance, but of course if the would stay centered, that would be a nice bonus.
